5.7/debian-10/rootfs/opt/bitnami/scripts/libmysql.sh

```diff
@@ -321,19 +321,21 @@ EOF
     fi
 }
 
+
 ########################
-# Run custom initialization scripts
+# Run custom scripts
 # Globals:
 #   DB_*
-# Arguments:
-#   None
+# Arguments: 
+#   $1 - 'init' or 'start' ('init' runs on first container start, 'start' runs everytime the container starts)
 # Returns:
 #   None
 #########################
-mysql_custom_init_scripts() {
-    if [[ -n $(find /docker-entrypoint-initdb.d/ -type f -regex ".*\.\(sh\|sql\|sql.gz\)") ]] && [[ ! -f "$DB_DATA_DIR/.user_scripts_initialized" ]] ; then
-        info "Loading user's custom files from /docker-entrypoint-initdb.d";
-        for f in /docker-entrypoint-initdb.d/*; do
+mysql_custom_scripts() {
+    if [[ -n $(find /docker-entrypoint-$1db.d/ -type f -regex ".*\.\(sh\|sql\|sql.gz\)") ]] && \
+      ([[ ! -f "$DB_DATA_DIR/.user_scripts_initialized" ]] || $1 == 'start'); then
```

Comment:
Although it is a long line, I think it is more readable in a single line:
```suggestion
    if [[ -n $(find /docker-entrypoint-"$1"db.d/ -type f -regex ".*\.\(sh\|sql\|sql.gz\)") ]] && { [[ ! -f "$DB_DATA_DIR/.user_scripts_initialized" ]] || [[ $1 = start ]]; } then
```

Apart from that, there are some changes fixing spellcheck linter issues:
- `docker-entrypoint-$1db.d` -> `docker-entrypoint-"$1"db.d`: https://github.com/koalaman/shellcheck/wiki/SC2086
- `&& (...); then` -> `&& { ...; } then`: https://github.com/koalaman/shellcheck/wiki/SC2235
- `|| $1 == 'start'` -> `[[ $1 = start ]]`: https://github.com/koalaman/shellcheck/wiki/SC2284